packages/expo-contacts/ios/ContactAccessButton/ContactAccessButtonView.swift

```diff
@@ -4,8 +4,8 @@ import ContactsUI
 import SwiftUI
 import ExpoModulesCore
 
-internal struct ExpoContactAccessButton: ExpoSwiftUI.View {
-  @EnvironmentObject
+internal struct ExpoContactAccessButton: ExpoSwiftUI.View, ExpoSwiftUI.WithHostingView {
+  @ObservedObject
```

Comment:
I like this change – imo we don't need to use context-like behavior to pass these props so this is way cleaner